lib/http.js

```diff
@@ -33,13 +33,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   const {routes} = cfg;
 
   app.post(routes.authenticate,
-    // FIXME add validation
-    // validate({body: 'VeresWalletCredential'}),
+    validate({body: 'vc-auth.login'}),
```

Comment:
The bedrock-validation system is a bit wonky, but the idea is this.  The schemas are namespaced by filename which is somewhat problematic when we're dealing with 100 different modules adding to the schemas.  Also, one wants to be able to quickly and easily determined where a schema has originated.  All of this to say, that if you would like this validator to be in a separate file, the filename should be something like `vc-issuer-auth` and it would be referenced as `vc-issuer-auth.login`.

Further, the schemas are added by referencing a folder.  This is why I had created a `subs` folder for some sub documents that are not exported in order to prevent polluting the namespace.

Reply:
I went ahead and just merged this into a single file here: https://github.com/digitalbazaar/bedrock-vc-issuer/pull/20/commits/61ad2df6fb5217e33da13e5552f80d2958dccf0f

We don't have enough validators yet to really need namespaces/ other modules.